etc/docker/Dockerfile.triton-jetpack-focal

```diff
@@ -96,7 +97,7 @@ RUN apt-get -y update && apt-get -y --no-install-recommends install \
 
 # Build grpc (and proto, etc.) from source, since we don't have a good version on focal
 RUN cd ${HOME}/opt/src && \
-    git clone --recurse-submodules -b v1.66.2 --depth 1 --shallow-submodules https://github.com/grpc/grpc && \
+    git clone --recurse-submodules -b v1.71.0 --depth 1 --shallow-submodules https://github.com/grpc/grpc && \
```

Comment:
1.72 is actually the newest gRPC release, but buf does not appear to have support for it yet, so use 1.71 for now